src/containers/cards/constants.js

```diff
@@ -1 +1,4 @@
+export const ADD_CARD = 'lists/ADD_CARD'
 export const GET_ALL_CARDS_IN_LIST = 'cards/GET_ALL_CARDS_IN_LIST'
+export const DELETE_CARD = 'cards/DELETE_CARD'
```

Comment:
Why do you have a `DELETE_CARD` and `SAVE_CARD_RANK` in this PR ? It's useless.

Reply:
Done